src/oca_github_bot/tasks/tag_needs_review.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) ACSONE SA/NV 2018
+# Distributed under the MIT License (http://opensource.org/licenses/MIT).
+
+from ..github import gh_call, repository
+from ..queue import getLogger, task
+
+_logger = getLogger(__name__)
+
+LABEL_NEEDS_REVIEW = "needs review"
+
+
+@task()
+def tag_needs_review(org, pr, repo, status, dry_run=False):
+    """On a successful execution of the CI tests, adds the `needs review`
+    label to the pull request if it doesn't have `wip:` at the
+    begining of the title (case insensitive). Removes the tag if the CI
+    fails.
+    """
+    with repository(org, repo) as gh_repo:
+        gh_pr = gh_call(gh_repo.pull_request, pr)
+        gh_issue = gh_pr.issue()
+        if status == "success" and not gh_pr.title.lower().startswith("wip:"):
+            if dry_run:
+                _logger.info(f"DRY-RUN add {LABEL_NEEDS_REVIEW} label")
```

Comment:
Use code py3.5 friendly

Reply:
@moylop260 I did it like this to keep it consistent with the other files:     
![image](https://user-images.githubusercontent.com/12100691/57196697-e56ad380-6f24-11e9-868e-f1a773169591.png)

and also, the pre-commit config file uses python 3.6:    
![image](https://user-images.githubusercontent.com/12100691/57196686-d4ba5d80-6f24-11e9-8aa1-40bbf52b1c3e.png)    

Should I change it to be python 3.5 compliant anyway?

Reply:
Ok
No problem
No changes required here then